react.md

```diff
@@ -4,55 +4,54 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. Here is a list of pros and cons to using the React library to build your application -- but some of them are false. Remove the false statements from the list:
```

Comment:
Close! The last false statement refers to React as a full stack framework. The key here is that React is only the V or view in an MVC framework. We have to handle the M and C with a different tool. But don't worry, we will dive deeper into that later on!